lib/services/locator.dart

```diff
@@ -1,12 +1,21 @@
 import 'package:dio/dio.dart';
+import 'package:flutter/material.dart';
 import 'package:get_it/get_it.dart';
+import 'package:hydrated_bloc/hydrated_bloc.dart';
+import 'package:path_provider/path_provider.dart';
 
 import '../api/api.dart';
 
 final locator = GetIt.instance;
 
-void registerServices() {
+Future<void> registerServices() async {
   const baseUrl = 'https://tazkrtak-api-demo.herokuapp.com/';
   final client = Dio(BaseOptions(baseUrl: baseUrl));
-  locator.registerLazySingleton(() => HelloService(client));
+
+  WidgetsFlutterBinding.ensureInitialized();
```

Comment:
This line should be in `main.dart`, it's related to UI, have nothing to do with services.